drivers/rawexec/driver.go

```diff
@@ -320,12 +321,16 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *cstru
 		return nil, nil, fmt.Errorf("failed to create executor: %v", err)
 	}
 
+	// If we are on linux, running as root, and cgroups aren't disabled by the
+	// operator - then use cgroups for pid management.
+	useCgroups := !d.config.NoCgroups && runtime.GOOS == "linux" && syscall.Geteuid() == 0
```

Comment:
Wonder how `syscall.Geteuid` behaves in Windows.  I made a helper function for this that might be handy.

Also the comment doesn't add much beyond the code itself.  Maybe comment why cgroups shouldn't be used for unprivileged users?

Reply:
In this case, the windows behaviour shouldn't matter too much (it'll be short circuited by runtime.GOOS not being linux).  I'll update the comment tho.